pom.xml

```diff
@@ -76,12 +76,12 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
+        <!--<dependency>
```

Comment:
Why is this commented out? Should it be deleted?

Reply:
yes, I commented it to test if that was removing the issue and it was, but forgot to remove it.

Reply:
I will remove it in my Java 17 Version because I've modified the pom so much that it will be a nightmare of merging it.